docs/usage.rst

```diff
@@ -3,19 +3,52 @@ Usage
 =====
 
 
-`ansible-rulebook` is either used as a command line tool or used as a web service with `ansible-rulebook-ui`.  
+`ansible-rulebook` is either used as a command line tool or used as a web service with `ansible-rulebook-ui`.
+
+
+======================
+CLI Usage and Examples
+======================
+
+
+The `ansible-rulebook` CLI supports the following options::
+
+    -h, --help                  Show this page
+    -v, --vars=<v>              Variables file
+    -i, --inventory=<i>         Inventory
+    --rules=<r>                 The rules file or rules from a collection
+    -S=<S>, --source_dir=<S>    Source dir
+    --vars=<v>                  A vars file
+    --env-vars=<e>              Comma separated list of variables to import
+                                from the environment
+    --redis_host_name=<h>       Redis host name
+    --redis_port=<p>            Redis port
+    --debug                     Show debug logging
+    --verbose                   Show verbose logging
+    --version                   Show the version and exit
+    --websocket-address=<w>     Connect the event log to a websocket
+    --id=<i>                    Identifier
+    --worker                    Enable worker mode
+    --project-tarball=<p>       Project tarball
 
 To get help from `ansible-rulebook` run the following::
 
     ansible-rulebook --help
 
+To check the version of `ansible-rulebook` run the following::
+
+    ansible-rulebook --version
+
 The normal method for running `ansible-rulebook` is the following::
 
     ansible-rulebook --inventory inventory.yml --rules rules.yml
 
-If you are using custom event source plugins use the following::
+.. note::
+    The `--rules` option requires the `--inventory` option.
```

Comment:
`--rules` flag has been changed to `--rulebook`
Ref: https://github.com/ansible/ansible-rulebook/pull/189